main.go

```diff
@@ -356,20 +363,22 @@ func processSource(nsCfg config.NamespaceConfig, t tail.Follower, parser *gonx.P
 			}
 		}
 
+		notCounterValues := relabeling.StripOnlyCounterValues(labelValues, relabelings)
```

Comment:
Could we maybe add an optizimation to call `StripOnlyCounterValues` if there actually _are_ "counter-only" values? This method basically copies the entire `labelValues` array (needlessly, if no counter-only values are configured) and is right on the critical path in the main processing loop.